runtime/local/src/lib.rs

```diff
@@ -760,7 +760,6 @@ impl pallet_vesting::Config for Runtime {
 parameter_types! {
     pub const DepositPerItem: Balance = deposit(1, 0);
     pub const DepositPerByte: Balance = deposit(0, 1);
-    pub const MaxValueSize: u32 = 16 * 1024;
```

Comment:
This is unused in local ? What's the default value ? I think we should keep it to ensure local test will be similar as test network

Reply:
It's old, unused constant. Dead code.